CONTRIBUTING.md

```diff
@@ -44,8 +44,8 @@ public class LongClassName {
 }
 ```
 
-Make sure to comment your code where possible.
+Make sure to add comments to your code where possible.
```

Comment:
Seems silly.

Reply:
I still don't understand your definition of "silly"

Reply:
I feel like "commenting your code" is used more in programming spaces as opposed to "adding comments to your code." Commenting is an act of itself.

Reply:
```suggestion
Make sure to comment your code where possible.
```